docusaurus/video/docusaurus/docs/api/basics/get_started.mdx

```diff
@@ -142,8 +151,21 @@ call.getOrCreate({
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+from getstream import Stream
+
+calltype_name = "default"
+call_request_data = {
+    "data": {
+        "created_by_id": "user_id",
+        "settings_override": {"audio": {"access_request_enabled": False}},
+    },
+    "members": [{"role": "speaker", "user_id": "user_id"}],
+}
+
+call_id = "your_call_id"
+
+client = Stream(api_key="your_api_key", api_secret="your_api_secret")
+response = client.video.get_or_create_call(call_type=calltype_name, call_id=call_id, data=call_request_data)
```

Comment:
@sachaarbonel it is probably nicer to have a `Call` class that captures the type/id params

```python
call = client.video.call("default", "id")
call.create()
call.start_broadcasting()
...
```

Reply:
it's done on the sdk side, updating the docs now